.buildkite/auditbeat/auditbeat-pipeline.yml

```diff
@@ -1,12 +1,18 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
 
 env:
-  IMAGE_UBUNTU_X86_64: "family/core-ubuntu-2204"
-  IMAGE_UBUNTU_ARM_64: "core-ubuntu-2004-aarch64"
+  IMAGE_UBUNTU_2004_X86_64: "family/core-ubuntu-2004"
+  IMAGE_UBUNTU_2204_X86_64: "family/core-ubuntu-2204"
+  IMAGE_UBUNTU_2004_ARM_64: "core-ubuntu-2004-aarch64"
+  IMAGE_UBUNTU_2204_ARM_64: "core-ubuntu-2204-aarch64"
+  IMAGE_DEBIAN_10: "family/core-debian-10"
+  IMAGE_DEBIAN_12: "family/core-debian-12"
+  IMAGE_CENTOS7: "family/core-centos-7"
+  IMAGE_SUSE15: "family/core-sles-15"
```

Comment:
maybe also add `family/core-sles-12` it is in the support matrix of `auditbeat`

Reply:
[Same](https://github.com/elastic/beats/pull/37892#discussion_r1481285263)